crates/orderbook/src/database/orders.rs

```diff
@@ -317,9 +319,9 @@ fn full_order_into_model_order(order: FullOrder) -> Result<Order> {
         ethflow_data,
         onchain_user,
         is_liquidity_order: class == OrderClass::Liquidity,
-        // TODO #643 when we add surplus fee caching, this will be properly stored
-        // in the db
-        surplus_fee: Default::default(),
+        surplus_fee: big_decimal_to_u256(&order.surplus_fee)
+            .ok_or_else(|| anyhow!("surplus_fee is not U256"))?,
```

Comment:
This can probably be `.context("surplus_fee is not U256")` instead of `.ok_or_else(|| anyhow!("surplus_fee is not U256"))`.

Reply:
I'll open a refactor PR tomorrow where I clean up all of these. I also have at least one other similar thing that I want to clean up.